cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -57,8 +57,14 @@ abstract class Configuration extends string {
   /** Holds if data flow through `node` is prohibited. */
```

Comment:
The QLDoc on `isBarrier` uses terminology that isn't easy to relate to the two new predicates. I suggest changing "through" to "into or out of".

Same thing with `isSanitizer`.